packages/xdl/src/project/publishAsync.ts

```diff
@@ -390,12 +394,22 @@ export async function publishAsync(
     });
   }
 
+  // Create project manifest URL
+  const url =
+    options.releaseChannel && options.releaseChannel !== 'default'
+      ? `${response.url}?release-channel=${options.releaseChannel}`
+      : response.url;
+
+  // Create project page URL
+  const websiteUrl = url.replace('exp.host', 'expo.io');
+  const splitUrl = websiteUrl.split('/');
+  splitUrl.splice(4, 0, 'projects');
+  const projectPageUrl = splitUrl.join('/');
```

Comment:
this feels a little brittle. we get the `url` from our `publish/new` endpoint: https://github.com/expo/expo-cli/blob/4f40c276ffe1c52f5bc13b914818ddc36646e460/packages/xdl/src/project/publishAsync.ts#L237

i imagine we should be changing the url on that endpoint rather than transforming it in expo-cli code

Reply:
Yeah I agree

Reply:
OK! This is merged https://github.com/expo/universe/pull/7395